README.md

```diff
@@ -45,27 +44,53 @@ You can also combine the flag options:
 
 ## Installation
 
-The current stable version of python-magic is available on PyPI and
-can be installed by running `pip install python-magic`.
+This module is a simple [CDLL](https://docs.python.org/3/library/ctypes.html) wrapper around the libmagic C library.
+The current stable version of python-magic is available on [PyPI](http://pypi.python.org/pypi/python-magic/)
+and can be installed by running `pip install python-magic`.
 
-Other sources:
+Compiled libmagic and the magic database come bundled in the wheels on PyPI.
+You can use your own `magic.mgc` database by setting the `MAGIC`
+environment variable, or by using `magic.Magic(magic_file='path/to/magic.mgc')`.
+If you want to compile your own libmagic, circumvent the wheels
+by installing from source: `pip install python-magic --no-binary python-magic`.
 
-- PyPI: http://pypi.python.org/pypi/python-magic/
-- GitHub: https://github.com/ahupp/python-magic
+For systems not supported by the wheels, pip installs from source,
+requiring libmagic to be available before installing python-magic:
 
-This module is a simple wrapper around the libmagic C library, and
-that must be installed as well:
+### Linux
 
-### Debian/Ubuntu
+The Linux wheels should run on most systems out of the box.
 
+Depending on your system and CPU architecture, there might be no compatible wheel uploaded.
+However, precompiled libmagic might still be available for your system:
+
+```sh
```

Comment:
It may be beneficial to add a library installation guide for SUSE as well

Reply:
can you provide the relevant command?

fwiw, I think mostly all linux flavours will be covered by the wheels in the PR description, so those users won't be needing the install from source instructions provided here.

Reply:
I guess that it would be 
`zypper install file-devel`
it might as well be required to do (in vivo test is required though)
`zypper install file-magic`

Currently I don't have OpenSUSE at my disposal for tests and it's likely that it would be a default package.